app/upgrades/v18/constants.go

```diff
@@ -0,0 +1,11 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package v18
+
+const (
+	// UpgradeName is the shared upgrade plan name for mainnet
+	UpgradeName = "v18.0.0"
+	// UpgradeInfo defines the binaries that will be used for the upgrade
+	UpgradeInfo = `'{"binaries":{"darwin/amd64":"https://github.com/evmos/evmos/releases/download/v18.0.0/evmos_18.0.0_Darwin_arm64.tar.gz","darwin/x86_64":"https://github.com/evmos/evmos/releases/download/v18.0.0/evmos_18.0.0_Darwin_x86_64.tar.gz","linux/arm64":"https://github.com/evmos/evmos/releases/download/v18.0.0/evmos_18.0.0_Linux_arm64.tar.gz","linux/amd64":"https://github.com/evmos/evmos/releases/download/v18.0.0/evmos_18.0.0_Linux_amd64.tar.gz","windows/x86_64":"https://github.com/evmos/evmos/releases/download/v18.0.0/evmos_18.0.0_Windows_x86_64.zip"}}'`
+)
```

Comment:
I think we can remove this going forward, I have always wondered why we keep this around but it hasn't been used ever before 😅

Reply:
we'll keep it for now. But we'll consider this for future releases